### ISSUE TITLE
gridle_get_column_width();

### ISSUE THREAD

**Issue by danielhq** [olivierbossel/gridle#66]
Hey

I was trying to use this function today:
.content__article {
	$width : gridle_get_column_width();
	padding: 0 $width;
}

but it gives me:
Undefined variable: "$context". 
on line 68 of gridle/_functions.scss

**Comment by pilaas** [comment]
I have similar problem:
```
[01:17:09] Error: Invalid null operation: "12 div null".
        on line 231 of /Users/me/Desktop/moje/gp/gp_server/src/gp/static_new/vendor/custom/gridle2/sass/_mixins.scss, in `_gridle'
(...)
```
This happens when I type: "@include gridle(12)"

**Comment by pferdefleisch** [comment]
@pilaas you're lucky... my error is `5 div null`

**Comment by pferdefleisch** [comment]
OK, the `5 div null` error is caused by gridle's point version update requiring a setup.
```sass
$settings : (
  context : 12,             // number of columns in your grid
  gutter-width : 20px,        // size of the gutters
  direction : ltr,             // direction
  name-multiplicator : 1     // to generate like grid-5, grid-10, etc...
  // and more... (see in advanced settings section)
);

@include gridle_setup($settings);
```

solved it for me.

**Comment by olivierbossel** [comment]
This has been fixed from 2.0.0 release.
